plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -159,12 +159,17 @@ class NewCreateServiceModalForm extends Component {
       return true;
     }
 
+    const isBaseConfigChanged = this.state.baseConfig !== nextState.baseConfig;
```

Comment:
@jfurrow @mlunoe could one of you please update to use `did` instead of `is`? I think it's very confusing when we mix present and past tense in a variable name.